index.bs

```diff
@@ -5140,8 +5140,8 @@ IANA "WebAuthn Extension Identifier" registry established by [[!WebAuthn-Registr
 - Specification Document: Section [[#sctn-uvm-extension]] of this specification
     <br/><br/>
 - WebAuthn Extension Identifier: credProps
-- Description: This [=client extension|client=] and [=authenticator extension|authenticator=] [=registration extension=] enables reporting of a newly-created [=credential=]'s properties to the 
-    calling [=[WRP]=]'s [=web application=].
+- Description: This [=client extension|client=] [=registration extension=] enables reporting of a newly-created [=credential=]'s properties,
+    as determine by the [=client=] to the calling [=[WRP]=]'s [=web application=].
```

Comment:
s/determine/determined/

Reply:
Also, I think we need a comma after `[=client=]`?

Reply:
yep, thanks :)

Reply:
Agreed and updated.